engine/execution/computation/computer/computer.go

```diff
@@ -354,71 +320,57 @@ func (e *blockComputer) executeCollection(
 				collection.Guarantee.CollectionID.String()))
 
 		e.log.Debug().
-			Hex("block_id", logging.Entity(collection.ctx.BlockHeader)).
+			Str("block_id", collection.blockId).
 			Hex("collection_id", logging.Entity(collection.Guarantee)).
 			Msg("executing collection")
 	}
 
 	defer colSpan.End()
 
 	for _, txBody := range collection.Transactions {
-		err := e.executeTransaction(txBody, colSpan, collectionView, collection.ctx, collectionIndex, txIndex, res, false)
+		err := e.executeTransaction(collection.blockId, txBody, colSpan, collectionView, collection.ctx, collectionIndex, txIndex, collector, collection.isSystemCollection)
```

Comment:
note that this was a bug (false should have been changed to collection.isSystemCollection) introduced in #3607.  This only impacted logging.